src/cpu-template-helper/src/template/dump/aarch64.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+use vmm::arch::aarch64::registers::RegSize;
```

Comment:
nit: commit message: `Updated internal representation of aarch64 registers`
Could give a brief explanation of what the new internal representation is.

Reply:
added additional explanation